kube/bundle.yaml

```diff
@@ -176,6 +176,7 @@ data:
         bootstrappers:
             - filesystem
             - commitlog
+            - uninitialized
```

Comment:
Ah much better name 👍

Reply:
Do we want to be putting peers after `commitlog` in all these configurations?

Reply:
haha I thought you wouldn't like the name but I couldn't think of anything better, and yeah sure why not. It should all work now